src/shared/components/views/VisualFeaturesView.tsx

```diff
@@ -70,9 +73,16 @@ function VisualFeaturesView<T>({
           params.accession,
           TabLocation.FeatureViewer
         )}
-        className="full-view"
+        title="View in the Feature Viewer"
```

Comment:
Accessibility issue when using an icon for a link, the browser doesn't know what it means and can't expose the information to screen readers. Here putting a title, it will be exposed to those tools.